### PR TITLE
magnetic pole

### PR DESCRIPTION
**Pull request by sgkang**
add magnetic pole (for magnetic dipole applet)

### CODE REVIEW

**Comment by sgkang** [comment]
hi @lheagy, can you review this? 
I am working on reviving ubc magnetic dipole applet,
and Doug wants to add magnetic pole.

**Comment by sgkang** [comment]
This is how it looks. 
image

**Comment by lheagy** [comment]
Very cool @sgkang! Would you mind creating a small example? 

I will look into the travis fail

**Comment by sgkang** [comment]
Hi @lheagy can you take a look travis error?
It seems error is occuring when examples are run in the docs, 
and it does not recognize `geoana.em.static.MagneticPoleWholeSpace`.

File: geoana/__init__.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 from . import em
 from . import utils
 
-__version__ = '0.0.4'
+__version__ = '0.0.6'
```
Comment (lheagy) [inline geoana/__init__.py]: @sgkang: it looks like the version number is out of sync here

File: setup.py [hunk 2]
```diff
@@ -31,7 +31,7 @@
 
 setup(
     name = 'geoana',
-    version = '0.0.4',
+    version = '0.0.3',
```
Comment (lheagy) [inline setup.py]: and here?

File: docs/conf.py [hunk 3]
```diff
@@ -64,9 +64,9 @@
 # built documents.
 #
 # The short X.Y version.
-version = u'0.0.4'
+version = u'0.0.6'
```
Comment (lheagy) [inline docs/conf.py]: is the version out of sync here?

**Comment by sgkang** [comment]
Hey @lheagy, I was messed up with versions, but fixed now. Can we merge this?

**Comment by lheagy** [comment]
@sgkang: Would you mind taking a look at the appveyor fail? it is passing on master, so something is up...

File: appveyor.yml [hunk 4]
```diff
@@ -39,7 +39,7 @@ install:
   - "conda create -q -n test-environment python=%PYTHON% numpy scipy matplotlib cython jupyter ipython pillow wheel"
   - activate test-environment
   - pip install -r requirements_dev.txt
-  - python setup.py install
+  - pip install git+git://github.com/simpeg/geoana.git@magnetic_pole
```
Comment (sgkang) [inline appveyor.yml]: @lheagy I am not sure why ..., but in appveyor it seem we are using geoana in pypi ... 
this passes the test

**Comment by lheagy** [comment]
image

https://ci.appveyor.com/project/lheagy/geoana/history

Hi @sgkang: let me know how this goes, if it is still giving you troubles, I can try to have a look tonight

**Pull request merged**